tlsr9/ble/stack/ble/B91/ble_config.h

```diff
@@ -61,7 +61,7 @@
 #elif(MCU_CORE_TYPE == MCU_CORE_827x)
 	#define	FIX_HW_CRC24_EN											0
 	#define HW_ECDH_EN                                      		1
-#elif(MCU_CORE_TYPE == MCU_CORE_9518)
```

Comment:
maybe better to leave original code and just redefine MCU_CORE_9518 into our MCU_CORE_B91 ?